build/dependencies.props

```diff
@@ -76,6 +76,7 @@
     <MicrosoftExtensionsWebEncodersPackageVersion>2.1.1</MicrosoftExtensionsWebEncodersPackageVersion>
 
     <!-- Packages that come from aspnet/Scaffolding. (Used only in the 2.1.x package archives.) -->
+    <!-- See src/Templating/build/dependencies.props for the versions used in templates. -->
```

Comment:
@ryanbrandenburg those versions are a bit older e.g. `$(MicrosoftVisualStudioWebCodeGenerationCorePackageVersion)` is `2.1.3-rtm-30913` instead of the `2.1.7` used here. Should we update that version in the release branches?

Reply:
I think I'm confused about where the `2.1.3-rtm-30913` version is coming from, so I don't know if we should update. Could you go into more detail?

Reply:
The `2.1.3-rtm-30913` version comes from `src/Templating/build/dependencies.props`. That file is the one the templating projects use, not this one.

Reply:
Ah, I missed that this was a PR against 2.1. I would assume that we should do 2.1.7 unless someone has reasons not to, but that just comes from a general desire to keep versions aligned rather than from specific knowledge.